.github/workflows/balena.yml

```diff
@@ -32,8 +32,9 @@ env:
     export-database \
     import-database \
     e2e-server-previous-dump
-
-  LIVECHAT_PORT: 443
+  INTEGRATION_JELLYFISH_APP_ID: jellyfish
+  INTEGRATION_JELLYFISH_APP_SECRET: changeme
```

Comment:
Can we drop these? I see we have defaults in `jellyfish-environment` set here: https://github.com/product-os/jellyfish-environment/blob/master/lib/variables/integration/defaults.ts#L18-L19